### PR TITLE
Fix rulers

### PR DESCRIPTION
**Pull request by predragnikolic**
closes https://github.com/sublimelsp/LSP-json/issues/180

### CODE REVIEW

**Comment by rchl** [comment]
description should also be updated

**Review by masterdigger — APPROVED** [review]
All in place!

**Review by masterdigger — APPROVED** [review]

File: schemas/preferences.sublime-settings.json [hunk 1]
```diff
@@ -217,7 +217,7 @@
           "default": "auto"
         },
         "rulers": {
-          "markdownDescription": "Columns in which to display vertical rulers.\n\nAs of ST 4133 column and style (see `ruler_style`) can be paired.\n\ne.g.: `\"rulers\": [[80, \"solid\"], [100, \"dotted\"]]`",
+          "markdownDescription": "Columns in which to display vertical rulers. Each entry may also be a\n\nlist of the column, the ruler style and optionally the ruler width, for\n\nexample: `[[80, \"solid\", 2], 78, [90, \"dotted\"]]`",
```
Comment (rchl) [inline schemas/preferences.sublime-settings.json:220]: We shouldn't be include newlines in the middle of the sentence. We should let it wrap by itself.

**Review by rchl — APPROVED** [review]

File: schemas/preferences.sublime-settings.json [hunk 2]
```diff
@@ -217,7 +217,7 @@
           "default": "auto"
         },
         "rulers": {
-          "markdownDescription": "Columns in which to display vertical rulers.\n\nAs of ST 4133 column and style (see `ruler_style`) can be paired.\n\ne.g.: `\"rulers\": [[80, \"solid\"], [100, \"dotted\"]]`",
+          "markdownDescription": "Columns in which to display vertical rulers. Each entry may also be a list of the column, the ruler style and optionally the ruler width, for example: `[[80, \"solid\", 2], 78, [90, \"dotted\"]]`",
```
Comment (rchl) [inline schemas/preferences.sublime-settings.json:220]: Not that if we want to separate paragraphs with `\n\n` then that's fine. I just meant hard wrapping at random point in the sentence.

**Pull request merged**